client/my-sites/checkout/checkout-thank-you/domain-transfer-to-any-user/index.tsx

```diff
@@ -16,31 +16,46 @@ interface DomainTransferToAnyUserContainerProps {
 const DomainTransferToAnyUser: React.FC< DomainTransferToAnyUserContainerProps > = ( {
 	domain,
 } ) => {
+	const upsellProps = {
+		title: translate( 'Professional email' ),
+		description: translate(
+			'85% of people trust an email address with a custom domain name over a generic one.'
+		),
+		meshColor: 'blue',
+		icon: emailImage,
+		action: (
+			<Button
+				href={ emailManagement( domain, domain ) }
+				onClick={ () =>
+					recordTracksEvent( 'calypso_domain_transfer_thank_you_professional_email_click' )
+				}
+			>
+				{ translate( 'Add email' ) }
+			</Button>
+		),
+	};
+
+	const products = [ <ProductDomain domainName={ domain } key="transferred-domain" /> ];
+
 	return (
-		<ThankYouLayout>
-			<DefaultThankYouHeader>
-				{ translate( 'Your domain transfer is underway' ) }
-			</DefaultThankYouHeader>
-			<DefaultSubHeader>
-				{ translate(
+		<Main className="is-redesign-v2">
+			<PageViewTracker
+				path="checkout/domain-transfer-to-any-user/thank-you/:domain"
+				title="Checkout Thank You"
+			/>
+
+			<MasterbarStyled canGoBack={ false } showContact={ true } />
+
+			<ThankYouV2
+				title={ translate( 'Your domain transfer is underway' ) }
+				subtitle={ translate(
 					'Domain transfers can take a few minutes, we’ll email you once it’s set up.'
 				) }
-			</DefaultSubHeader>
-			<ProductDomain domain={ domain } />
-			<DomainTransferToAnyUserFooter />
-			<DefaultUpsell
-				title={ translate( 'Professional email' ) }
-				description={ translate(
-					'85% of people trust an email address with a custom domain name over a generic one.'
-				) }
-				meshColor="blue"
-				icon={ emailImage }
-				href={ emailManagement( domain, domain ) }
-				buttonText={ translate( 'Add email' ) }
-				trackEvent="calypso_domain_transfer_thank_you_professional_email_click"
+				products={ products }
+				footerDetails={ getDomainFooterDetails() }
+				upsellProps={ upsellProps }
 			/>
-			<ConfettiAnimation delay={ 1000 } />
```

Comment:
We're removing the confetti animation: https://github.com/Automattic/wp-calypso/pull/86186#discussion_r1447167198